examples/adversarial/test_hans.py

```diff
@@ -25,13 +25,10 @@
 
 import numpy as np
 import torch
-from torch.utils.data import DataLoader, RandomSampler, SequentialSampler, TensorDataset
```

Comment:
not related to this PR, but in the next maybe we can rename this file to `run_hans.py` like other examples